html/resources/files/app/tagline/tagline-off-ios.json

```diff
@@ -139,6 +139,7 @@
             "message" : "Help translate Open Food Facts into Filipino"
         }
     },
+  // TODO
```

Comment:
JSON does not support comments

Reply:
it won't be in the final PR, it's a memento to finish this PR